### PR TITLE
For #9782: Clean up unused FeatureFlags

### PR DESCRIPTION
**Pull request by hakkikaancaliskan**
I've currently found out this ones unused.
I've removed webPushIntegration because looks like all fatal bugs solved as it said from note.

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/components/FxaServer.kt [hunk 1]
```diff
@@ -6,30 +6,23 @@ package org.mozilla.fenix.components
...
-    const val REDIRECT_URL = "https://accounts.firefox.com/oauth/success/$CLIENT_ID"
+    private const val CLIENT_ID = "a2270f727f45f648"
 
-    @Suppress("ConstantConditionIf", "UNUSED_PARAMETER")
-    fun redirectUrl(context: Context) = if (FeatureFlags.asFeatureWebChannelsDisabled) {
-        REDIRECT_URL
-    } else {
-        "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
-    }
+    fun redirectUrl() = "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/components/FxaServer.kt:18]: This should be a constant
```suggestion
    const val REDIRECT_URL = "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
```

Comment (hakkikaancaliskan) [reply app/src/main/java/org/mozilla/fenix/components/FxaServer.kt]: Wait :) it's used in somewhere as function.

Comment (hakkikaancaliskan) [reply app/src/main/java/org/mozilla/fenix/components/FxaServer.kt]: https://github.com/mozilla-mobile/fenix/blob/36b1c710e7cf2612a8d28585f301ec1140381710/app/src/main/java/org/mozilla/fenix/components/Services.kt#L29
Should we hardcode it or go with function?

Comment (hakkikaancaliskan) [reply app/src/main/java/org/mozilla/fenix/components/FxaServer.kt]: Done.

**Comment by liuche** [comment]
Opening this as #11659 to run CI, so you can follow along there.

**Pull request closed without merging**